<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YaRXP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="toolButton_L1">
        <property name="text">
         <string>L1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_L2">
        <property name="text">
         <string>L2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_L3">
        <property name="text">
         <string>L3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>Events</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="tileset_scroll">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>296</width>
             <height>287</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="TilesetWidget" name="tileset_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>256</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="map_tree_widget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Klagenfurt</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Viktring</string>
           </property>
           <item>
            <property name="text">
             <string>Player House</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Route 91</string>
           </property>
           <item>
            <property name="text">
             <string>Haider Denkmal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gate House</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="map_scroll">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>474</width>
           <height>487</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="MapWidget" name="map_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionLayer1"/>
    <addaction name="actionLayer2"/>
    <addaction name="actionLayer3"/>
    <addaction name="actionEvents"/>
    <addaction name="separator"/>
    <addaction name="actionDim_other_Layers"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionCurrent_Layers_and_below"/>
    <addaction name="actionAll_Layers"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="actionPen"/>
    <addaction name="actionRectangle_Fill"/>
    <addaction name="actionFlood_Fill"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuDraw"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLayer1">
   <property name="text">
    <string>Layer&amp;1</string>
   </property>
   <property name="toolTip">
    <string>Layer1</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionLayer2">
   <property name="text">
    <string>Layer&amp;2</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionLayer3">
   <property name="text">
    <string>Layer&amp;3</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="text">
    <string>&amp;Events</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionDim_other_Layers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Dim other Layers</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionCurrent_Layers_and_below">
   <property name="text">
    <string>&amp;Current Layers and below</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionAll_Layers">
   <property name="text">
    <string>&amp;All Layers</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="text">
    <string>Pen</string>
   </property>
  </action>
  <action name="actionRectangle_Fill">
   <property name="text">
    <string>Rectangle Fill</string>
   </property>
  </action>
  <action name="actionFlood_Fill">
   <property name="text">
    <string>Flood Fill</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilesetWidget</class>
   <extends>QLabel</extends>
   <header>tilesetwidget.h</header>
  </customwidget>
  <customwidget>
   <class>MapWidget</class>
   <extends>QLabel</extends>
   <header>mapwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
